iree/compiler/Dialect/Flow/Transforms/DispatchConfig.cpp

```diff
@@ -179,15 +181,19 @@ OpDispatchPolicy::FusionType OpDispatchPolicy::fuseOutput(Operation *anchorOp,
 // TODO(b/144530470): replace with tablegen attributes/interfaces.
 bool OpDispatchPolicy::isUnsupportedFusionOp(Operation *op) {
   return isa<mhlo::ConcatenateOp, mhlo::ConvOp, mhlo::DotGeneralOp, mhlo::DotOp,
-             mhlo::PadOp, mhlo::ReduceOp, mhlo::ReduceWindowOp,
-             mhlo::TorchIndexSelectOp>(op) ||
-         isRootOnlyOp(op);
+             mhlo::ReduceOp, mhlo::ReduceWindowOp, mhlo::TorchIndexSelectOp>(
+             op) ||
+         isRootOnlyOp(op) || isLeafOnlyOp(op);
 }
 
 bool OpDispatchPolicy::isRootOnlyOp(Operation *op) {
   return isa<mhlo::SliceOp>(op);
 }
 
+bool OpDispatchPolicy::isLeafOnlyOp(Operation *op) {
```

Comment:
I would expect that `SliceOp` also be moved into `isLeafOnlyOp`. Would be good to split this change to refactor the `OpDispatchPolicy` to either add `isLeafOnlyOp` or rename `isRootOnlyOp` to `isLeafOnlyOp` and then evaluate if the dispatch region creation logic still works.

Reply:
They are different, one is leaf only and another is root only. I plan to rename them in a later patch since this is still in progress.